src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -168,6 +170,53 @@ public BuildScriptSnippet GenerateBashBuildScriptSnippet(BuildScriptGeneratorCon
             };
         }
 
+        private void GetAppOutputDirPath(dynamic packageJson, Dictionary<string, string> buildProperties)
```

Comment:
nit: this method is actually getting the value but also setting it in the build properties. Should we split this in two? Or rename it as it's not called `Get`?

Reply:
Sure I agree. Will change it in a different PR.